### PR TITLE
Add docstrings for qudit_gates.py in css

### PR DESCRIPTION
**Pull request by singular-value**

### CODE REVIEW

File: cirq-superstaq/cirq_superstaq/ops/qudit_gates.py [hunk 1]
```diff
@@ -504,9 +516,13 @@ def qubit_subspace_op(
 QutritZ2 = QutritZ2PowGate()
 
 
-def custom_resolver(  # pylint: disable=missing-function-docstring
+def custom_resolver(
     cirq_type: str,
 ) -> Optional[Type[cirq.Gate]]:
+    """Used by deserialization code to convert string representations of gates to Cirq objects.
```
Comment (vtomole) [inline cirq-superstaq/cirq_superstaq/ops/qudit_gates.py:522]: Let's coordinate on how to make this docstring match with https://github.com/SupertechLabs/client-superstaq/pull/513/files#r1227291598 for consistency

**Review by vtomole — COMMENTED** [review]
Document `Returns`. See internal dev guide on our docstring standard

**Comment by singular-value** [comment]
> Document what the functions and methods return via `Returns`. See internal dev guide on our docstring standard

Thanks, much appreciated. Reading it now.

File: cirq-superstaq/cirq_superstaq/ops/qudit_gates.py (hunk 1 above)
Comment (singular-value) [reply cirq-superstaq/cirq_superstaq/ops/qudit_gates.py]: @vtomole I tried to pull from (IMO) the best of both of ours. PTAL.

**Review by dowusu-antwi — APPROVED** [review]
LGTM.

**Pull request merged**